canonical-websites/www.canonical.com
# Design feedback for /careers

## Done

Address feedback in https://github.com/ubuntudesign/vanilla-squad/issues/56

## QA

- Pull code
- Run `./run serve --watch`
- Check http://0.0.0.0:8002/careers against the checklist above

## Issue / Card

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/56
